101-bundle-json.md

```diff
@@ -238,8 +251,7 @@ Fields:
 - images: The list of dependent images
   - `description`: The description field provides additional context of the purpose of the image. 
   - `imageType`: The `imageType` field MUST describe the format of the image. The list of formats is open-ended, but any CNAB-compliant system MUST implement `docker` and `oci`. The default is `oci`.
-  - `image`: The REQUIRED `image` field provides a valid reference (REGISTRY/NAME:TAG) for the image. Note that SHOULD be a CAS SHA, not a version tag as in the example above.
-  - `digest`: MUST contain a digest, in [OCI format](https://github.com/opencontainers/image-spec/blob/master/descriptor.md#digests), to be used to compute the integrity of the image. The calculation of how the image matches the digest is dependent upon image type. (OCI, for example, uses a Merkle tree while VM images are checksums.)
+  - `image`: The REQUIRED `image` field provides a valid reference (`REGISTRY/NAME:TAG@DIGEST`) for the image. Note that SHOULD be a CAS SHA, not a version tag as in the example above. See the section Resolving Image References for more information on how this is to be resolved.
```

Comment:
_ Note that SHOULD be a CAS SHA, not a version tag as in the example above. _

This is a bit unclear.

Reply:
> Note that SHOULD be a CAS SHA, not a version tag as in the example above.

This should be dropped. Meaning of the digest is explained in Resolving Image References